### PR TITLE
Order creation and sad path testing

### PR DESCRIPTION
**Pull request by fentontaylor**
- Clicking Place Order on the new order form creates an order in the database with the customer's info and brings you to a page where you can review the details of your order.
- If the new order form is not completely filled out, the user sees a flash message and is redirected to the order form page

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-class ItemsController<ApplicationController
+class ItemsController < ApplicationController
```
Comment (alect47) [inline app/controllers/items_controller.rb]: classic

Comment (fentontaylor) [reply app/controllers/items_controller.rb]: haha, you know me

File: app/controllers/orders_controller.rb [hunk 2]
```diff
@@ -5,7 +5,26 @@ def new
   end
 
   def create
-    
+    order = Order.new(order_params)
+    if order.save
```
Comment (alect47) [inline app/controllers/orders_controller.rb]: I like the use of .save

File: spec/features/orders/new_spec.rb [hunk 3]
```diff
@@ -56,3 +60,79 @@
...
+    click_link 'Add to Cart'
+    visit "/items/#{dog_bone.id}"
+    click_link 'Add to Cart'
+    visit "/items/#{dog_bone.id}"
+    click_link 'Add to Cart'
+    visit '/cart'
+    click_link 'Checkout'
+
+    name = "Ian Douglas"
+    address = '123 Testing Pl.'
```
Comment (alect47) [inline spec/features/orders/new_spec.rb]: Hahahah

**Pull request merged**